lighthouse-core/test/computed/metrics/lantern-interactive-test.js

```diff
@@ -10,8 +10,11 @@ const assert = require('assert');
 
 const trace = require('../../fixtures/traces/progressive-app-m60.json');
 const devtoolsLog = require('../../fixtures/traces/progressive-app-m60.devtools.log.json');
+const iframeTrace = require('../../fixtures/traces/iframe.trace.json');
```

Comment:
can you add the chrome version this was captured in to the filename? i rather like that